src/Compilers/VisualBasic/Portable/Symbols/PreprocessingSymbol.vb

```diff
@@ -13,10 +13,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         Implements IPreprocessingSymbol
 
         Private ReadOnly _name As String
+        Private ReadOnly _assembly As IAssemblySymbol
+        Private ReadOnly _module As IModuleSymbol
```

Comment:
remove these.

Reply:
Same reason as above in C#

Reply:
same reason as c#.  we shoudn't need these :)